src/dev/build/tasks/os_packages/docker_generator/templates/base/Dockerfile

```diff
@@ -189,7 +189,7 @@ ENTRYPOINT ["/bin/tini", "--"]
 CMD ["/app/kibana.sh"]
 # Generate a stub command that will be overwritten at runtime
 RUN mkdir /app && \
-    echo -e '#!/bin/bash\nexec /usr/local/bin/kibana-docker' > /app/kibana.sh && \
+    /usr/bin/echo -e '#!/bin/bash\nexec /usr/local/bin/kibana-docker' > /app/kibana.sh && \
```

Comment:
This is for local testing and can be ignored, it's meant to be overwritten.  `RUN` uses sh, which has a builtin echo that doesn't support the `-e` flag.